ports/stm32/boards/NUCLEO_H7A3ZI/stm32h7xx_hal_conf.h

```diff
@@ -0,0 +1,20 @@
+/* This file is part of the MicroPython project, http://micropython.org/
+ * The MIT License (MIT)
+ * Copyright (c) 2019 Damien P. George
+ */
+#ifndef MICROPY_INCLUDED_STM32H7XX_HAL_CONF_H
+#define MICROPY_INCLUDED_STM32H7XX_HAL_CONF_H
+
+#include "boards/stm32h7xx_hal_conf_base.h"
+
+// Oscillator values in Hz
+//#define HSE_VALUE (24000000)
```

Comment:
Please remove commented-out lines.

What is the HSE value?

Reply:
It is 8MHz. Should be ok now.